### ISSUE TITLE
vmagent metric_relabel_configs affect automatically generated timeseries 

### ISSUE THREAD

**Issue by weenxin** [VictoriaMetrics/VictoriaMetrics#592]
**Describe the bug**
According [prometheus document](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs) ,

 `Metric relabeling does not apply to automatically generated timeseries such as up.`

**To Reproduce**
some config with `metric_relabel_configs`  with `keep` action(not include `up` metrics).

**Expected behavior**
the `up` metrics should not be removed.

**Version**
```
$ ./victoria-metrics-prod --version
vmagent-20200628-065703-heads-master-0-g05d754d-dirty-ff8a2fbe
```

**Used command-line flags**

```
bin/vmagent  \
-httpListenAddr=:8429 \
-memory.allowedPercent=70  \
-maxInsertRequestSize=1024000000 \
-promscrape.config=bin/prometheus.yaml \
-promscrape.maxScrapeSize=10240000000 \
-promscrape.configCheckInterval=1m \
-remoteWrite.tmpDataPath=data \
-remoteWrite.queues=50 \
-remoteWrite.maxBlockSize=1024000000 \
-remoteWrite.flushInterval=10s \
-remoteWrite.url=http://host1:8480/insert/1128/prometheus/api/v1/write  \
-remoteWrite.url=http://host2:8480/insert/1128/prometheus/api/v1/write  \
-remoteWrite.url=http://host3:8480/insert/1128/prometheus/api/v1/write  \
-remoteWrite.url=http://host4:8480/insert/1128/prometheus/api/v1/write
```

**Comment by valyala** [comment]
The issue should be fixed in the commit 1cd01b5359511452d2b10aa318c4e18d2cafb5d9 . Leaving the issue open until the bugfix is included in the next release.

**Comment by valyala** [comment]
The bugfix has been included in [v1.37.4](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
